control/reset.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+#
+# Copyright (c) 2019, CESAR. All rights reserved.
+#
+# SPDX-License-Identifier: BSD 3-Clause
+
+import logging
+import os
+import json
+
+from pkg_resources import resource_string
+from mongo import MongoConn
+from rabbitmq import RabbitMQConn
+
+RESET_SETTINGS_FILE = 'resetSettings.json'
+
+def createReset():
+    """
+        A factory to create a instance of Reset
+    """
+    json_file = resource_string(__name__, RESET_SETTINGS_FILE)
+    settings = json.loads(json_file)
+    mongodb_settings = settings.get('mongodb')
+    mongo_conn = MongoConn(mongodb_settings.get('host'),
+                            mongodb_settings.get('port'))
+    rabbitmq_conn = RabbitMQConn()
+
+    return Reset(mongo_conn, rabbitmq_conn)
+
+class Reset():
+    """
+    Control KNoT Gateway state using a dbus implementation
```

Comment:
that is not what the class is doing. It is implementing the features the dbus services will call.